actions/utils/logging.ts

```diff
@@ -0,0 +1,99 @@
+import assert = require("assert");
+import winston = require("winston");
+const { Loggly } = require("winston-loggly-bulk");
+
+let logger: undefined | any;
+
+export function initLogging(
+  logglyToken: string,
+  tags: string[],
+  logOnlyIfError = false
+) {
+  const newLogger = new Loggly({
+    token: logglyToken,
+    subdomain: "cowprotocol",
+    tags,
+    json: true,
+  });
+
+  if (logger !== undefined) {
+    // We had a previous logger, remove it and add the new one
+    winston.remove(logger);
+    winston.add(newLogger);
+    logger = newLogger;
+    return;
```

Comment:
Why returning when it already exists?
Is the rest of the setup not needed?

Reply:
I just do this to change the tags. The rest uses the `winston.log` which use the new logger.

Reply:
One execution could init the logging multiple times, this is why i did it